cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -811,6 +813,47 @@ def _validate_sparsity(self):
                         f"to use this type of matrix for the given sparsity."
                     )
 
+    def _validate_seurat_convertibility(self):
+        """
+        Use length of component matrices to determine if the anndata object will be unable to be converted to Seurat by
+        virtue of the R language's array size limit (4-byte signed int length). Add warning for each matrix which is
+        too large.
+        rtype: None
+        """
+        to_validate = [self.adata.X]
+        to_validate_name = ["X"]
+        # check if there's raw data
+        if self.adata.raw:
+            to_validate.append(self.adata.raw.X)
+            to_validate_name.append("raw")
+        failing_matrix_names: List[str] = []
+        # Check length of component arrays
+        for x, x_name in zip(to_validate, to_validate_name):
+            if (
```

Comment:
I think you can use https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.issparse.html

Reply:
I don't know if there are certain types of sparse matrices that we don't want. Not familiar enough with technical details; I took this from the `_validate_sparsity` method immediately above.

Reply:
For our purposes though you're right. This should be fine.